src/app/pages/contact/embedded.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Form from './form';
+import './embedded.scss';
+
+// File is js because it is dynamically imported (and minimal)
+export default function EmbeddedForm() {
+    const [submitted, setSubmitted] = React.useState(false);
+
+    if (submitted) {
+        return <ThankYou />;
+    }
+
+    return (
+        <main id="maincontent" className="embedded-contact">
+            <h1>Report an issue</h1>
+            <div>
+                Need to suggest a content correction instead? Fill out the{' '}
+                <a href="/errata" target="_blank">
+                    errata form
+                </a>
+                .
+            </div>
+            <Form setSubmitted={setSubmitted} />
+        </main>
+    );
+}
+
+function ThankYou() {
+    const signalDone = React.useCallback(
+        () => window.parent.postMessage('contact form submitted', '*'),
```

Comment:
We use screaming case snake for these in assignable, so it would be nice if they all match.

Reply:
So `CONTACT_FORM_SUBMITTED`, then. I'll need to change it in both PRs.